src/jsmodules/array.js

```diff
@@ -1,7 +1,7 @@
 const firstDisplay = [
-  { task: 'Wash the dishes', bool: true, id: 1 },
-  { task: 'Build a To-Do app', bool: true, id: 2 },
-  { task: 'dont\' forget about the webpack', bool: true, id: 3 },
+  { task: 'Wash the dishes', bool: true, id: 0 },
+  { task: 'Build a To-Do app', bool: true, id: 1 },
+  { task: 'dont\' forget about the webpack', bool: true, id: 2 },
```

Comment:
- [ ] I suggest you remove these hardcoded items as required of us in the project requirements. This `firstDisplay` variable has been imported in the `index.js` file, which means you'll have to take care of removing the importation too. The project requirement for this is quoted below:

> **- Remove all hardcoded items from the tasks array.**

Reply:
Removed all hardcoded items from the task array.